<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>627</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>5</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(170, 0, 0);
background-color: rgb(170, 0, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="win_btn">
    <property name="geometry">
     <rect>
      <x>-9</x>
      <y>230</y>
      <width>531</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 2px;
background-color: rgb(197, 158, 229);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 39px;
color: rgb(0, 145, 0);
background-color: rgb(170, 85, 255);</string>
    </property>
    <property name="text">
     <string>ü§öüèª</string>
    </property>
   </widget>
   <widget class="QPushButton" name="front_btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>490</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 39px;
color: rgb(0, 145, 0);
background-color: rgb(170, 85, 255);</string>
    </property>
    <property name="text">
     <string>‚úãüèª</string>
    </property>
   </widget>
   <widget class="QPushButton" name="player_score_btn">
    <property name="geometry">
     <rect>
      <x>141</x>
      <y>431</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>Your Score : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="com1_score">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>Computer1 Score : </string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>301</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border :2px ;
border-color: rgb(170, 0, 127);
border-style : dashed;
border-radius: 38px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="player_choice">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>43</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border :2px solid purple;
border-style : dashed;
border-radius: 38px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>79</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border :2px ;
border-color: rgb(170, 0, 127);
border-style : dashed;
border-radius: 38px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="comp1_choice">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>43</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border :2px solid purple;
border-style : dashed;
border-radius: 38px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>79</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border :2px ;
border-color: rgb(170, 0, 127);
border-style : dashed;
border-radius: 38px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="comp2_choice">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>43</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border :2px solid purple;
border-style : dashed;
border-radius: 38px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="comp2_score">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>Computer2 Score : </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>504</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
